class_names: ["oneway", "highwayentrance", "stopsign", "roundabout", "park", "crosswalk", "noentry", "highwayexit", "priority", "lights", "block", "pedestrian", "car"]
# confidence_thresholds: [0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.85, 0.9]
# confidence_thresholds: [0.7, 0.6, 0.7, 0.7, 0.7, 0.7, 0.7, 0.65, 0.65, 0.65, 0.65, 0.7, 0.75]
confidence_thresholds: [0.7, 0.6, 0.7, 0.7, 0.7, 0.7, 0.7, 0.65, 0.65, 0.65, 0.65, 0.7, 0.55]
max_distance_thresholds: [1.0, 1.5, 2.0, 2.0, 2.4, 1.5, 1.0, 0.9, 1.5, 1.5, 1.5, 1., 1.4]
counter_thresholds: [2,3,3,3,3,3,3,3,3,3,3,2,3]
ncnn_model: "sissi753-opt"
# v8_model: "citycocov2lgtclab_20"
v8_model: "0517track20"
